data/docs-redirects.js

```diff
@@ -180,9 +180,15 @@ export const docsRedirects = {
   '/docs/cloud/api/variable-sets.html': '/cloud-docs/api-docs/variable-sets',
   '/docs/cloud/api/variables.html': '/cloud-docs/api-docs/variables',
   '/docs/cloud/api/vcs-events.html': '/cloud-docs/api-docs/vcs-events',
-  '/docs/cloud/api/workspace-resources.html': '/cloud-docs/api-docs/workspace-resources',
-  '/docs/cloud/api/workspace-variables.html': '/cloud-docs/api-docs/workspace-variables',
-  '/docs/cloud/api/workspaces.html': '/cloud-docs/api-docs/workspaces',
+  '/docs/cloud/api/workspaces/index.html': '/cloud-docs/api-docs/workspaces/index',
+  '/docs/cloud/api/worksapces/workspace-crud-actions.html': '/cloud-docs/api-docs/workspaces/workspace-crud-actions',
+  '/docs/cloud/api/worksapces/workspace-lock-mechanism.html': '/cloud-docs/api-docs/workspaces/workspace-lock-mechanism',
+  '/docs/cloud/api/worksapces/workspace-remote-state-consumers.html': '/cloud-docs/api-docs/workspaces/workspace-remote-state-consumers',
+  '/docs/cloud/api/worksapces/workspace-resources.html': '/cloud-docs/api-docs/workspaces/workspace-resources',
+  '/docs/cloud/api/worksapces/workspace-ssh-key-assignment.html': '/cloud-docs/api-docs/workspaces/workspace-ssh-key-assignment',
```

Comment:
you've got a couple lines here where there is a typo with "worksapces" :-)

Reply:
🤦‍♀️ Thanks!